data/maps/BattleFrontier_Lounge3/scripts.inc

```diff
@@ -1,214 +1,218 @@
-	goto BattleFrontier_Lounge3_EventScript_261DAF
+	goto_if_ge BattleFrontier_Lounge3_EventScript_CheckBetResults
+	goto BattleFrontier_Lounge3_EventScript_AskToEnterChallenge
 	end
 
-BattleFrontier_Lounge3_EventScript_261F12:: @ 8261F12
+BattleFrontier_Lounge3_EventScript_CheckBetResults:: @ 8261F12
 	compare VAR_FRONTIER_GAMBLER_STATE, FRONTIER_GAMBLER_PLACED_BET
-	goto_if_eq BattleFrontier_Lounge3_EventScript_261F9E
+	goto_if_eq BattleFrontier_Lounge3_EventScript_ChallengeNotAttempted
 	compare VAR_FRONTIER_GAMBLER_STATE, FRONTIER_GAMBLER_WON
-	goto_if_eq BattleFrontier_Lounge3_EventScript_261F2E
-	goto BattleFrontier_Lounge3_EventScript_261F71
+	goto_if_eq BattleFrontier_Lounge3_EventScript_WonChallenge
+	goto BattleFrontier_Lounge3_EventScript_LostChallenge
 	end
 
-BattleFrontier_Lounge3_EventScript_261F2E:: @ 8261F2E
-	msgbox BattleFrontier_Lounge3_Text_263334, MSGBOX_DEFAULT
+BattleFrontier_Lounge3_EventScript_WonChallenge:: @ 8261F2E
+	msgbox BattleFrontier_Lounge3_Text_HelloChampHeresYourPoints, MSGBOX_DEFAULT
 	compare VAR_FRONTIER_GAMBLER_AMOUNT_BET, FRONTIER_GAMBLER_BET_5
-	call_if_eq BattleFrontier_Lounge3_EventScript_261F80
+	call_if_eq BattleFrontier_Lounge3_EventScript_RewardBet5
 	compare VAR_FRONTIER_GAMBLER_AMOUNT_BET, FRONTIER_GAMBLER_BET_10
-	call_if_eq BattleFrontier_Lounge3_EventScript_261F8A
+	call_if_eq BattleFrontier_Lounge3_EventScript_RewardBet10
 	compare VAR_FRONTIER_GAMBLER_AMOUNT_BET, FRONTIER_GAMBLER_BET_15
-	call_if_eq BattleFrontier_Lounge3_EventScript_261F94
-	msgbox BattleFrontier_Lounge3_Text_2633D4, 9
+	call_if_eq BattleFrontier_Lounge3_EventScript_RewardBet20
+	msgbox BattleFrontier_Lounge3_Text_ObtainedBattlePoints, 9
 	special GiveFrontierBattlePoints
-	msgbox BattleFrontier_Lounge3_Text_2633F2, MSGBOX_DEFAULT
+	msgbox BattleFrontier_Lounge3_Text_ThinkOfMeForAnotherChallenge, MSGBOX_DEFAULT
 	setvar VAR_FRONTIER_GAMBLER_STATE, FRONTIER_GAMBLER_WAITING
 	release
 	end
 
-BattleFrontier_Lounge3_EventScript_261F71:: @ 8261F71
-	msgbox BattleFrontier_Lounge3_Text_263298, MSGBOX_DEFAULT
+BattleFrontier_Lounge3_EventScript_LostChallenge:: @ 8261F71
+	msgbox BattleFrontier_Lounge3_Text_NiceTryCantReturnPoints, MSGBOX_DEFAULT
 	setvar VAR_FRONTIER_GAMBLER_STATE, FRONTIER_GAMBLER_WAITING
 	release
 	end
 
-BattleFrontier_Lounge3_EventScript_261F80:: @ 8261F80
-	buffernumberstring 0, 10
-	setvar VAR_0x8004, 10
+BattleFrontier_Lounge3_EventScript_RewardBet5:: @ 8261F80
+	buffernumberstring 0, (BET_AMOUNT_5 * 2)
+	setvar VAR_0x8004, (BET_AMOUNT_5 * 2)
 	return
 
-BattleFrontier_Lounge3_EventScript_261F8A:: @ 8261F8A
-	buffernumberstring 0, 20
-	setvar VAR_0x8004, 20
+BattleFrontier_Lounge3_EventScript_RewardBet10:: @ 8261F8A
+	buffernumberstring 0, (BET_AMOUNT_10 * 2)
+	setvar VAR_0x8004, (BET_AMOUNT_10 * 2)
 	return
 
-BattleFrontier_Lounge3_EventScript_261F94:: @ 8261F94
-	buffernumberstring 0, 30
-	setvar VAR_0x8004, 30
+BattleFrontier_Lounge3_EventScript_RewardBet20:: @ 8261F94
```

Comment:
Should be `RewardBet15` instead of 20, right?

Reply:
Whoops, yes. They're 5, 10, 15